src/server/admin/cmds/cmds.go

```diff
@@ -55,5 +57,22 @@ func Cmds(noMetrics *bool) []*cobra.Command {
 		}),
 	}
 	restore.Flags().StringVarP(&url, "url", "u", "", "An object storage url (i.e. s3://...) to restore from.")
-	return []*cobra.Command{extract, restore}
+	inspectCluster := &cobra.Command{
+		Use:   "inspect-cluster",
+		Short: "Returns info about the pachyderm cluster",
+		Long:  "Returns info about the pachyderm cluster",
+		Run: cmdutil.RunFixedArgs(0, func(args []string) error {
+			c, err := client.NewOnUserMachine(metrics, "user")
+			if err != nil {
+				return err
+			}
+			ci, err := c.InspectCluster(context.Background(), nil)
+			if err != nil {
+				return err
```

Comment:
This is missing a `ScrubGRPC` call. We generally put those in `src/client/...` though so I think you should add `InspectCluster` to `src/client/admin.go`, put the `ScrubGRPC` call in there and call it from here.